app/views/mortgage_calculator/transaction_tax_calculator/_next_home_buyer_explanation.html.erb

```diff
@@ -0,0 +1,38 @@
+<p><%= I18n.t("land_transaction_tax.how_calculated") %></p>
+<p><%= I18n.t("land_transaction_tax.how_calculated_additional") %></p>
+
+<table class="mortgagecalc__table stamp-duty__table">
+  <thead>
+    <tr>
+      <th class="mortgagecalc__table__price">
+        <%= I18n.t("land_transaction_tax.table.property_price_header") %>
```

Comment:
either that or you need to refactor this to 
`<%= I18n.t("#{i18n_locale_namespace}.table.property_price_header") %>`

Reply:
Good spot. I already fixed ♻️. After this fix I run the commands: `grep -r "I18n.t('land_and" app/views` and `grep -r "I18n.t('land_tra" app/views` and they didn't return anything, so I think now is clear.

Reply:
nice! and both `land_transaction_tax` and `land_and...` have all the translation keys that `I18n.t("#{i18n_locale_namespace}` applies to them?

Reply:
Yes, they do. And the dummy app is setup to raise errors if locale does not exist.